fetch/api/request/request-reset-attributes.https.html

```diff
@@ -33,4 +37,39 @@
       }
     }
   }, 'Request.isReloadNavigation is reset with non-empty RequestInit');
+
+promise_test(async (t) => {
+    const scope = 'resources/hello.txt?name=isHistoryNavigation';
+    let frame;
+    let reg;
+
+    try {
```

Comment:
It seems this test would pass if something throws before you hit an assert. I'd prefer it if this didn't have the try/catch wrapper.

Reply:
This is "try-finally", not "try-catch", so I think exceptions will be treated as failures.